CONTRIBUTING.md

```diff
@@ -15,3 +15,24 @@
 - Do not use camel case or underscores for folder names.
 - Arrange your folders in alphabetical order.
 - Folder name or folder description should not contain special characters.
+- 
+## Add an Extension
+
+1. Edit `cmd/populator/modules.go` under `func init() `  with the extension details.
+```
+{
+    name:        "liquibase-commercial-dynamodb",
+    category:    Extension,
+    url:         "https://repo1.maven.org/maven2/org/liquibase/ext/liquibase-commercial-dynamodb",
+    artifactory: Maven{},
+}
+```
+2. Edit `internal/app/packages.json` with the extension details.
+```
+{
+    "name": "extension-name",
+    "category": "extension", // or "driver"
```

Comment:
this can also be `pro` and the categories should match with the populator (which is an enum not a string)